test/SimpleStore.ts

```diff
@@ -65,16 +65,47 @@ export class SimpleStore extends StoreBase {
         return this._subscribeWithEnumKeyData[TriggerKeys.First] + this._subscribeWithEnumKeyData[TriggerKeys.Second];
     }
 
+    @autoSubscribeWithKey(TriggerKeys.First)
+    getSingleKeySingleASKey(@key id: string): number {
+        return this._get(id) + this._subscribeWithEnumKeyData[TriggerKeys.First];
+    }
+
+    @autoSubscribeWithKey([TriggerKeys.First, TriggerKeys.Second])
+    getSingleKeyMultiASKey(@key id: string): number {
+        return this._get(id) + this._subscribeWithEnumKeyData[TriggerKeys.First] + this._subscribeWithEnumKeyData[TriggerKeys.Second];
+    }
+
+    @autoSubscribeWithKey(TriggerKeys.First)
+    getMultiKeySingleASKey(@key id: string, @key id2: string): number {
+        return this._get(id) + this._get(id2) + this._subscribeWithEnumKeyData[TriggerKeys.First];
+    }
+
+    @autoSubscribeWithKey([TriggerKeys.First, TriggerKeys.Second])
```

Comment:
There are no examples of multiple `@key` without `@autoSubscribeWithKey`, so that case is untested.